<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lugdulo'v</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/lugdulov.qrc">
    <normaloff>:/res/icon.png</normaloff>:/res/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>Stations:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="stationsComboBox">
      <item>
       <property name="text">
        <string>Auto</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0" rowspan="2" colspan="2">
     <widget class="QCommandLinkButton" name="searchLinkButton">
      <property name="text">
       <string>Search Stations</string>
      </property>
      <property name="icon">
       <iconset resource="../res/lugdulov.qrc">
        <normaloff>:/res/slot.png</normaloff>:/res/slot.png</iconset>
      </property>
      <property name="description">
       <string>Search stations by name, check their status and bookmark them.</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QCommandLinkButton" name="bookmarkLinkButton">
      <property name="text">
       <string>Favorites Stations</string>
      </property>
      <property name="icon">
       <iconset resource="../res/lugdulov.qrc">
        <normaloff>:/res/favorites.png</normaloff>:/res/favorites.png</iconset>
      </property>
      <property name="description">
       <string>Manage your favorites stations and check their status quickly</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QCommandLinkButton" name="mapLinkButton">
      <property name="text">
       <string>Maps</string>
      </property>
      <property name="icon">
       <iconset resource="../res/lugdulov.qrc">
        <normaloff>:/res/map.png</normaloff>:/res/map.png</iconset>
      </property>
      <property name="description">
       <string>Show nearby stations on a Map.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Lugdulo'V</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../res/lugdulov.qrc">:/res/icon.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>318</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="quitAction"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutQtAction"/>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="aboutQtAction">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/lugdulov.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>quitAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
